test/linter/test-versions.ts

```diff
@@ -159,91 +144,103 @@ const checkVersions = (
-          // If the data is to be mirrored, make sure it is mirrorable
-          if (!browsers[browser].upstream) {
-            logger.error(
-              chalk`{bold ${browser}} is set to mirror, however {bold ${browser}} does not have an upstream browser.`,
-            );
-          }
+      for (const property of ['version_added', 'version_removed']) {
+        const version = statement[property];
+        if (property == 'version_removed' && version === undefined) {
+          // Undefined is allowed for version_removed
           continue;
         }
-
-        for (const property of ['version_added', 'version_removed']) {
-          const version = statement[property];
-          if (property == 'version_removed' && version === undefined) {
-            // Undefined is allowed for version_removed
-            continue;
-          }
-          if (!isValidVersion(browser, category, version)) {
-            logger.error(
-              chalk`{bold ${property}: "${version}"} is {bold NOT} a valid version number for {bold ${browser}}\n    Valid {bold ${browser}} versions are: ${validBrowserVersions[
-                browser
-              ].join(', ')}`,
-              { tip: browserTips[browser] },
-            );
-          }
+        if (!isValidVersion(browser, category, version)) {
+          logger.error(
+            chalk`{bold ${property}: "${version}"} is {bold NOT} a valid version number for {bold ${browser}}\n    Valid {bold ${browser}} versions are: ${Object.keys(
+              browsers[browser],
+            ).join(', ')}`,
+            { tip: browserTips[browser] },
+          );
         }
 
-        if ('version_added' in statement && 'version_removed' in statement) {
-          if (statement.version_added === statement.version_removed) {
-            logger.error(
-              chalk`{bold version_added: "${statement.version_added}"} must not be the same as {bold version_removed} for {bold ${browser}}`,
-            );
-          }
+        if (typeof version === 'string' && version.includes('≤')) {
+          const releaseData =
+            browsers[browser].releases[version.replace('≤', '')];
           if (
-            typeof statement.version_added === 'string' &&
-            typeof statement.version_removed === 'string' &&
-            addedBeforeRemoved(statement) === false
+            !releaseData ||
+            !releaseData.release_date ||
+            new Date(releaseData.release_date) > twoYearsAgo
           ) {
             logger.error(
-              chalk`{bold version_removed: "${statement.version_removed}"} must be greater than {bold version_added: "${statement.version_added}"}`,
+              chalk`{bold ${property}: "${version}"} is {bold NOT} a valid version number for {bold ${browser}}\n    Ranged values are only allowed for browser versions released two years or older. Ranged values are also not allowed for browser versions without a known release date.}`,
```

Comment:
Extra `}` broke the template:

```suggestion
              chalk`{bold ${property}: "${version}"} is {bold NOT} a valid version number for {bold ${browser}}\n    Ranged values are only allowed for browser versions released two years or older. Ranged values are also not allowed for browser versions without a known release date.`,
```